src/ui/templates/results/resultsheader.hbs

```diff
@@ -14,11 +14,17 @@
   {{#if (and _config.showResultCount resultsCount)}}
     <div class="yxt-ResultsHeader-resultsCount" aria-label="{{resultsCountStart}} through {{resultsCountEnd}} of {{resultsCount}}">
       <div aria-hidden="true">
```

Comment:
Not specific to this change, but why is this aria-hidden=true? It seems like this is always shown and information for assistive technology. 

Should this still be true if we are allowing this template to be changed as config?

Reply:
I think we will want a class to target the innermost parent of either custom resultsCountText or default. That way if we need to make a change that affects both, we can using their most direct parent. Is there a reason we have this classless aria-hidden div?

Reply:
Basically, we wanted to override the screen reader text. The goal was to have it only read the custom text we gave it (the aria label) and ignore the text in the spans. What's the best way to do this?

Reply:
It seems aria-label does not do anything for the div element (as screen-readers focus on interactable elements not static elements)

Reply:
What we have now SGTM, I would be careful about classless divs, though, for easy parent targeting.